README.md

```diff
@@ -23,24 +22,16 @@ There are additional methods:
 
 ## Usage
 
-### Contract
+### Environment setup
 
-Set the following env vars:
+Copy the `.env.example` to `.env` and set the applicable configuration variables for the testing / deployment environment.
 
-```
-RECEIVER
-TO_TOKEN
-KEEPER
-SETTLEMENT
-APP_DATA
-TARGET_SAFE
-MIN_OUT
-```
+### Deploy module
 
-Then run the script to deploy the module:
+Run the script to deploy the module:
 
 ```
-forge script ./script/COWFeeModule.s.sol \
+forge script ./script/DeployCOWFeeModule.s.sol \
```

Comment:
I decided to rename the script to use a verb. This script deploys the module, so I suggested that name